buildspec.yaml

```diff
@@ -6,6 +6,11 @@ phases:
       java: openjdk11
     commands:
       - pip3 install aws-sam-cli
+  pre_build:
```

Comment:
build in codebuild fails because of non-ASCII characters in some hardcoded test data